doc/content/bib/articles_using_tmap8.bib

```diff
@@ -1,3 +1,14 @@
+@article{FRANKLIN2025115128,
```

Comment:
```suggestion
@article{Franklin2025,
```

Let's make this a bit more easy to digest when it could be used in the text somewhere.